Dockerfile.plugin.e2e

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/playwright:v1.47.1
+FROM mcr.microsoft.com/playwright:v1.48.0
```

Comment:
Not related to this PR but the build failed:

```
    ╔══════════════════════════════════════════════════════════════════════╗
    ║ Looks like Playwright Test or Playwright was just updated to 1.48.0. ║
    ║ Please update docker image as well.                                  ║
    ║ -  current: mcr.microsoft.com/playwright:v1.47.1-jammy               ║
    ║ - required: mcr.microsoft.com/playwright:v1.48.0-jammy               ║
    ║                                                                      ║
    ║ <3 Playwright Team                                                   ║
    ╚══════════════════════════════════════════════════════════════════════╝

```

Reply:
Reminder, to update: https://github.com/grafana/explore-profiles/blob/main/e2e/README.md#the-build-of-my-pr-has-failed-because-playwright-was-just-updated-how-to-fix-it